config/proto/probe.proto

```diff
@@ -52,16 +52,19 @@ message HermesProbeDef {
   TargetSystem target_system = 3;
   ProbeType probe_type = 4;
 
+  // Default = 3600
```

Comment:
Place these default comments as inline comments. They are harder to miss this way.

Reply:
I've done so across all files.